recipe/meta.yaml

```diff
@@ -21,7 +20,7 @@ requirements:
     - python
     - pip
     - cython
-    - numpy 1.12
```

Comment:
Should the `host` NumPy be dependent on the python version, so that we're building against the oldest possible NumPy ABI?

Reply:
We don't have numpy 1.12 built for Python 3.7. I was lazy here and bumped it to 1.14 but ideally we should build a 1.12 for Python 3.7.

Reply:
Indeed. I'm happy with whatever gets this working for the RC :)